src/mqt/yaqs/core/methods/stochastic_process.py

```
@@ -169,8 +169,7 @@ def stochastic_process(
         ValueError: If a 2-site jump is not nearest-neighbor, or if the jump operator does not act on 1 or 2 sites.
     """
     dp = calculate_stochastic_factor(state)
-    rng = np.random.default_rng()
-    if noise_model is None or rng.random() >= dp:
+    if noise_model is None or np.random.random() >= dp:  # noqa: NPY002
```

Ruff (0.15.0)

[warning] Unused `noqa` directive (non-enabled: `NPY002`)

Remove unused `noqa` directive

(RUF100)